.github/workflows/build-system.yml

```diff
@@ -29,44 +30,49 @@ jobs:
           name: Windows - Visual Studio 2022
         - os: macos-11
           name: macOS 11 - Apple-Clang 13
-          configuration: Debug
-        - os: macos-11
-          name: macOS 11 - Apple-Clang 13
-        exclude:
-        - os: macos-11
-          configuration: RelWithDebInfo
 
     steps:
       - name: Checkout Code
         uses: actions/checkout@v2
 
-      - name: Install Vulkan SDK (Linux / macOS)
-        uses: humbletim/install-vulkan-sdk@v1.1.1
```

Comment:
Isn't this version too old?

Reply:
macOS vendors it with its own script, and Linux comes preinstalled from the docker image so this is no longer needed (although the version there matches the Action, not the SDK version 😄 )